package.json

```diff
@@ -18,6 +18,10 @@
   "browser": {
     "./dist-esm/request/defaultAgent.js": "./dist-esm/request/defaultAgent.browser.js"
   },
+  "files": [
+    "dist",
+    "dist-esm"
```

Comment:
shouldn't we be distributing our license as well?

Reply:
LICENSE automatically gets picked up by npm no matter what you configure